metrics/cgroups/v2/io.go

```diff
@@ -0,0 +1,136 @@
+// +build linux
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package v2
+
+import (
+	v2 "github.com/containerd/containerd/metrics/types/v2"
+	metrics "github.com/docker/go-metrics"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var ioMetrics = []*metric{
+	{
+		name: "io_major",
```

Comment:
major/minor values should be handled as in v1?
https://github.com/containerd/containerd/blob/b9fad5e310fafb453def5f1e7094f4c36a9806d2/metrics/cgroups/v1/blkio.go#L128

Reply:
We have different format for IoStats in `cgroups.v2` so it's not possible to make it 100% similar, do you think this format is not readable?

Reply:
This does not make sense to me as a metric.  I don't understand why this isn't a label

Reply:
the io read/write are metrics labeled for a device with a major/minor, the major/minor are not metrics